### PR TITLE
Dictionary lab

### PR DESCRIPTION
**Pull request by derizor**
I am not sure if output for line 77 and 82 is correct.

### CODE REVIEW

**Comment by derizor** [comment]
Mailroom part 2 is still in progress

File: students/Deniss_Semcovs/Lesson04/dict_lab.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+print(s4)
+#Display if s3 is a subset of s2 and if s4 is a subset of s2
+print("Is s3 a subset of s2?")
+s3.issubset(s2)
+print("Is s4 subset of s2?")
+s4.issubset(s2)
+
+#Create a set with the letters in ‘Python’ and add ‘i’ to the set.
+print("Sets2")
+wPython = set()
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/dict_lab.py:75]: Don't need this line

File: students/Deniss_Semcovs/Lesson04/dict_lab.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+wPython = set({"p","y","t","h","o","n"})
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/dict_lab.py:76]: Don't need the parentheses

File: students/Deniss_Semcovs/Lesson04/dict_lab.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+print(wPython)
+wPython.add("i")
+print(wPython)
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/dict_lab.py:79]: This certainly works - also wPython should be snake_case, and in meaningful terms, like my_set for example

File: students/Deniss_Semcovs/Lesson04/dict_lab.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+#Create a frozenset with the letters in ‘marathon’.
+wMarathon = frozenset({"m","a","r","a","t","h","o","n"})
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/dict_lab.py:81]: Same feedback for this section as well.

File: students/Deniss_Semcovs/Lesson04/dict_lab.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+print(wMarathon)
+#Display the union and intersection of the two sets.
+print(wPython.intersection(wMarathon))
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/dict_lab.py:84]: Intersection would be the combination of the two sets, so:
my_intersection = frozenset.intersection(wPython, wMarathon)

File: students/Deniss_Semcovs/Lesson04/mailroom_part2.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+data = [
+    ["Tom Hanks", 1000, 500],
+    ["Leonardo DiCaprio", 300, 1200, 500],
+    ["Natalie Portman", 5000],
+    ["Harrison Ford", 500, 2000, 100],
+    ["Scarlett Johansson", 1500, 1000],
+    ["Jennifer Aniston", 2000, 2000, 2000],
+]
```
Comment (charlesrobison) [inline students/Deniss_Semcovs/Lesson04/mailroom_part2.py:9]: Need to use a dictionary for your user database in this version of mailroom.  Please revise and resubmit for another review.

**Pull request merged**